### PR TITLE
CON-229: modularize avail node content

### PR DESCRIPTION
**Pull request by DannyS03**
- Removes tabs from Avail node content and creates doc for each section
- General edits

### CODE REVIEW

File: docs/avail/how-tos/avail-quick-start.md [hunk 1]
```diff
@@ -22,12 +22,12 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
 # How to use Polygon Avail
 
-:::note 
+:::note
 
 We are working on improving many of the current features. We appreciate you using our testnet and encourage your valuable feedback through one of our **[community channels](https://polygon.technology/community/)**.
```
Comment (ashucoder9) [inline docs/avail/how-tos/avail-quick-start.md:27]:
```suggestion
We are working on improving many of the current features. We appreciate you using our testnet and encourage your valuable feedback through one of our [<ins>community channels</ins>](https://polygon.technology/community/).
```

File: docs/avail/how-tos/avail-quick-start.md [hunk 2]
```diff
@@ -161,45 +161,45 @@ the transaction details.
...
-We have abstracted out the nitty-gritty of verifying data availability and have hosted a light client 
+We have abstracted out the nitty-gritty of verifying data availability and have hosted a light client
 for your use. All you need to do is click on the block number against your desired transaction and
-see all of the block details. 
+see all of the block details.
 
-You will also see a *confidence factor*. If it shows `0%`, give it some time and recheck it later. 
-Otherwise, it should show a non-zero confidence level indicating the probability with which the underlying data 
-is available. 
+You will also see a *confidence factor*. If it shows `0%`, give it some time and recheck it later.
```
Comment (ashucoder9) [inline docs/avail/how-tos/avail-quick-start.md:183]:
```suggestion
You will also see a **confidence factor**. If it shows `0%`, give it some time and recheck it later.
```

File: docs/avail/node/avail-light-client-da.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
+---
+id: avail-light-client-da
+title: Build and Run a Light Client with Data Availability
+sidebar_label: Run a Light Client
+description: "Learn about running DA."
```
Comment (ashucoder9) [inline docs/avail/node/avail-light-client-da.md:5]:
```suggestion
description: Learn about building and running a Light Client with Data Availability
```

File: docs/avail/node/avail-light-client-da.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+First, build the Docker images, `client:asdr` (using branch `feature/app-specific-data-retrieval_2`) and `da:asdr`
+(using branch `feature/app-specific-data-retrieval`):
+
+```shell
+export DOCKER_BUILDKIT = 1
+docker build --ssh default -t client:asdr --build-arg BRANCH=feature/app-specific-data-retrieval_2 -f images/client/Dockerfile images/client/
+```
+
+Next, run the services using **docker-compose.light-client.yml**:
```
Comment (ashucoder9) [inline docs/avail/node/avail-light-client-da.md:28]:
```suggestion
Next, run the services using `docker-compose.light-client.yml`:
```

File: docs/avail/node/avail-light-client-da.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+```shell
+export DOCKER_BUILDKIT=1
+docker build -t da:ava-33  --build-arg BRANCH=miguel/ava-33-create-monk-template-for-da-testnet -f images/da/Dockerfile images/da/
+```
+
+### Step 2: Load Monk templates
+
+The testnet only need to load two monk templates:
+
+- **monk/polygon-da-base.matic.today.yaml**, which contains common definition for DevNet & TestNet.
```
Comment (ashucoder9) [inline docs/avail/node/avail-light-client-da.md:51]:
```suggestion
- `monk/polygon-da-base.matic.today.yaml`, which contains common definitions for DevNet & TestNet.
```

File: docs/avail/node/avail-light-client-da.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+- **monk/polygon-da-devnet.matic.today.yaml**, where validators are defined.
```
Comment (ashucoder9) [inline docs/avail/node/avail-light-client-da.md:52]:
```suggestion
- `monk/polygon-da-devnet.matic.today.yaml`, where validators are defined.
```

File: docs/avail/node/avail-validator.md [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+
+```shell
+docker run -p 30333 --name my_val 0xpolygon/avail:latest --in-peers=10 --node-key 80027666cebec66464611eb0d5c36416213d83a9c689006a80efcf479826de7d
+```
+
+This image uses two volumes:
+  - `/da/state` to store the database of the chain
+  - `/da/keystore` to store the validator's private keys
+
+Most likelihood you want to bind these volumes to a specific points, like:
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:63]:
```suggestion
Most likely you want to bind these volumes to a specific point, like:
```

File: docs/avail/node/avail-validator.md [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+root@5f55e51e5a85:/da# /da/bin/data-avail key insert \
+      --chain=/da/genesis/testnet.chain.spec.raw.json \
+      --base-path=/da/state/ \
+      --keystore-path=/da/keystore/ \
+      --suri=0x7d98...cae6 \
+      --key-type=babe \
+      --scheme=Sr25519
+```
+
+The **--suri** parameter is the private key as a mnemonic phrase (or secret phrase) where you can generate
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:88]:
```suggestion
The `--suri` parameter is the private key as a mnemonic phrase (or secret phrase) where you can generate
```

File: docs/avail/node/avail-validator.md [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+one using the `subkey` tool in Substrate.
+
+:::note Learn about subkey
+
+To learn about how to use subkey, visit the [Subkey Substrate documentation](https://docs.substrate.io/v3/tools/subkey/).
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:93]:
```suggestion
To learn about how to use the subkey, visit the [<ins>Subkey Substrate documentation</ins>](https://docs.substrate.io/v3/tools/subkey/).
```

File: docs/avail/node/avail-validator.md [hunk 10]
```diff
@@ -0,0 +1,211 @@
...
+| audi     | Sr25519   |
+
+## Bond AVL tokens
+
+It is highly recommended that you set up a stash and controller account and have separate key (two separate accounts) for both.
+
+:::info Stash and Controller Keys
+
+- A controller key is used to control staking actions for your account
+- A stash key is used to control your funds. **It is recommended that the stash key be a cold wallet or offline and not be used for account related activities like submitting extrinsics.
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:113]:
```suggestion
- A stash key is used to control your funds. It is recommended that the stash key be a cold wallet or offline and not be used for account-related activities like submitting extrinsics.
```

File: docs/avail/node/avail-validator.md [hunk 11]
```diff
@@ -0,0 +1,211 @@
...
+
+ - Bond the AVL of the Stash account. These token will be put at stake for the security of the network and
+   subject to slashing.
+ - Select the Controller. This is the account that will decide when to start or stop validating.
+
+First, go to the **Developer** tab in the [Avail Apps](https://devnet-avail.polygon.technology/)
+navbar and click on **Extrinsics**.
+
+* **Stash** account - Select your Stash account. In this example, we bond 1001 AVL tokens, where the
+  minimum bonding amount is 1000. Make sure that your Stash account contains at least this much.
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:144]:
```suggestion
  minimum bonding amount is 1000. Make sure that your Stash account contains at least this many tokens.
```

File: docs/avail/node/avail-validator.md [hunk 12]
```diff
@@ -0,0 +1,211 @@
...
+Save this result for a later step.
+
+You can restart your node at this point.
+
+### Submitting the `setKeys` transaction
+
+You need to tell the chain your Session keys by signing and submitting an extrinsic. This is what associates
+your validator with your Controller account.
+
+Navigate to the [Network > Staking](https://devnet-avail.polygon.technology/#/staking).
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:197]:
```suggestion
Navigate to the [**Network &rarr; Staking**](https://devnet-avail.polygon.technology/#/staking).
```

File: docs/avail/node/avail-light-client-da.md (hunk 6 above)
Comment (DannyS03) [reply docs/avail/node/avail-light-client-da.md:52]: Would advise keeping these bolded as they aren't syntax or a param.

File: docs/avail/node/avail-validator.md [hunk 13]
```diff
@@ -0,0 +1,211 @@
...
+Here, you can perform various staking actions. Navigate to **Account actions** , and select **Set Session Key**
```
Comment (ashucoder9) [inline docs/avail/node/avail-validator.md:198]:
```suggestion
Here, you can perform various staking actions. Navigate to **Account actions**, and select **Set Session Key**
```

File: docs/avail/node/build-data-availability.md [hunk 14]
```diff
@@ -0,0 +1,100 @@
+---
+id: avail-data-avail
+title: Avail Validator Data Availability
+sidebar_label: Data Availability
+description: "Learn about running DA."
```
Comment (ashucoder9) [inline docs/avail/node/build-data-availability.md:5]:
```suggestion
description: Learn about running an Avail Validator node for Data Availability.
```

File: docs/avail/node/build-data-availability.md [hunk 15]
```diff
@@ -0,0 +1,100 @@
...
+
+Only build the node binaries
+
+```shell
+cargo build --release -p data-avail
+```
+
+## Optional: How to generate deterministic WASM
+
+:::note This step is **not required** and it should only be used to verify that WASM matches with
```
Comment (ashucoder9) [inline docs/avail/node/build-data-availability.md:34]:
```suggestion
:::note

This step is **not required** and it should only be used to verify that WASM matches with
```

**Review by ashucoder9 — APPROVED** [review]

**Comment by DannyS03** [comment]
thanks for reviewing the docs themselves @ashucoder9!

**Pull request merged**